src/bundle/Controller/ContentOnTheFlyController.php

```diff
@@ -91,7 +91,7 @@ public function createContentAction(
             }
         }
 
-        return new ContentCreateOnTheFlyView('@ezdesign/content/content_on_the_fly/content_create_on_the_fly.html.twig', [
+        return new ContentCreateOnTheFlyView('@ezdesign/ui/on_the_fly/content_create_on_the_fly.html.twig', [
```

Comment:
`Content on the fly` is invented name, so we should use `content_on_the_fly` here